### ISSUE TITLE
[8.0] Migration to v8 fails with project_issue post migration

### ISSUE THREAD

**Issue by mariuszmizgier** [OCA/OpenUpgrade#108]
As with the latest code I've tried to migrate my development database (which is really small one with just a few records in each module) - after resolving most of the issues thanks to the solutions posted here, I came across one, which is really tough. Here is a part of the log related to that:

```
2014-10-13 06:12:01,393 6594 ERROR dev_migrated openerp.modules.loading: Error executing post migration script for module project_issue
: wrap() takes exactly 1 argument (2 given)
2014-10-13 06:12:01,395 6594 CRITICAL dev_migrated openerp.service.server: Failed to initialize database `dev_migrated`.
Traceback (most recent call last):
  File "/var/tmp/openupgrade/8.0/server/openerp/service/server.py", line 901, in preload_registries
registry = RegistryManager.new(dbname, update_module=update_module)
  File "/var/tmp/openupgrade/8.0/server/openerp/modules/registry.py", line 324, in new
openerp.modules.load_modules(registry._db, force_demo, status, update_module)
  File "/var/tmp/openupgrade/8.0/server/openerp/modules/loading.py", line 387, in load_modules
force, status, report, loaded_modules, update_module, upg_registry)
  File "/var/tmp/openupgrade/8.0/server/openerp/modules/loading.py", line 295, in load_marked_modules
loaded, processed = load_module_graph(cr, graph, progressdict, report=report, skip_modules=loaded_modules, perform_checks=perform_checks, upg_registry=upg_registry)
  File "/var/tmp/openupgrade/8.0/server/openerp/modules/loading.py", line 215, in load_module_graph
migrations.migrate_module(package, 'post')
  File "/var/tmp/openupgrade/8.0/server/openerp/modules/migration.py", line 179, in migrate_module
mod.migrate(self.cr, pkg.installed_version)
TypeError: wrap() takes exactly 1 argument (2 given)
```

Any ideas how to approach that? Help on that would be greatly appreciated

**Comment by red15** [comment]
@mariuszmizgier Try the fixes from PR #128 , I just ran into these issues myself.

Note: since the storing of the priority may have already been lost it's best to restart from a clean database and run the migration from scratch.

**Comment by pedrobaeza** [comment]
Solved in mentioned PR, so I close this.
